### DISCUSSION TITLE
source-files-FRITZ.Repeater_1200_AX-07.30.tar.gz

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#541] [Allgemein]
Es gibt was neues
http://osp.avm.de/fritzwlan/fritzrepeater-1200-ax/

**Comment by fda77** [comment]
Am besten machst du source-links immer hier rein: https://github.com/Freetz-NG/freetz-ng/discussions/134
Pro Link 1 Eintrag, damit man sie einzeln verstecken kann
